tests/login-shell/session-variables.nix

```diff
@@ -0,0 +1,19 @@
+{ pkgs, lib, ... }:
+{
+  imports = [
+    ./base.nix
+    "${builtins.fetchTarball "https://github.com/nix-community/home-manager/archive/master.tar.gz"}/nixos"
```

Comment:
We shouldn't fetch home-manager from master. This can unexpectedly break.

Also why do we even use home-manager here? We can set session variables with NixOS, too.

Reply:
I used home-manager for the test, because that was specifically what someone had problems with. I'll have to take a look at how sessionVariables work in NixOS

Reply:
I've changed the URL to download the release version corresponding to the current NixOS version. We can't use `environment.sessionVariables`, because, as it turns out, those are set in `/etc/set-environment` which is also sourced outside of a login-shell thus defeating the purpose of the test